students/hyska_monika/lesson_06_dicts_tuples_sets_args_kwargs/ListToDictionaryFunctionForFantasyGameInventory.py

```diff
@@ -0,0 +1,17 @@
+# function to add items to dictionary
+from FantasyGameInventory import display_inventory
+
+
+def add_to_inventory(inventory, added_items):
+    for item in added_items:
+        if item in inventory:
```

Comment:
This is the place you could use `setdefault` instead of four lines:
`inventory.setdefault(item, 0) += 1`

Reply:
Sorry, it turns out it doesn't work... you would need two lines:
```
inventory.setdefault(item, 0)
inventory[item] += 1

Reply:
And another way would be: `inventory[item] = inventory.get(item, 1)`